src/Data/Text/Internal/Private.hs

```diff
@@ -11,7 +11,8 @@
 
 module Data.Text.Internal.Private
     (
-      runText
+      addrLen
```

Comment:
Just export `addrLen` from `Data.Text.Show` and import it in `Data.Text.Foreign` please.

Reply:
You got it.

Reply:
I meant to revert `Data.Text.Internal.Private` back: do not move `addrLen` outside of `Data.Text.Show`, just export it from there directly.

Reply:
Yep, I understood.